### ISSUE TITLE
 'spacy.tokens.doc.Doc' object has no attribute 'string' when extracting direct quotations

### ISSUE THREAD

**Issue by clhl** [chartbeat-labs/textacy#181]
Hi,

I am trying to extract direct quotations from my data with this code:

```
doc = textacy.Doc(tweets.decode('utf-8'))
test_ext = textacy.extract.direct_quotations(doc)
list1 = list(test_ext)
print list1

```

But I'm seeing the error: `AttributeError: 'spacy.tokens.doc.Doc' object has no attribute 'string'`.

**Comment by bdewilde** [comment]
Hi @clhl , this got fixed by PR #177 , and the fix is in already master.

**Comment by clhl** [comment]
Thank you!
